tools/sil-opt/SILOpt.cpp

```diff
@@ -426,8 +426,10 @@ int main(int argc, char **argv) {
     runSILLoweringPasses(*CI.getSILModule());
   } else {
     auto *SILMod = CI.getSILModule();
-    {
-      auto T = irgen::createIRGenModule(SILMod, getGlobalLLVMContext());
+    { const auto PSPs = CI.getPrimarySpecificPathsForAtMostOnePrimary();
```

Comment:
Missing newline here!

Reply:
Fixed! Great to have worked together and figured out how to do this right. Thanks!